client/src/configs/env.ts

```diff
@@ -1,3 +1,7 @@
+export const DEFAULT_LANGUAGE = process.env.REACT_APP_DEFAULT_LANGUAGE || 'English';
+
+export const REACT_APP_LANGUAGE_SWITCHER = process.env.REACT_APP_LANGUAGE_SWITCHER === 'true';
```

Comment:
This should be `export const LANGUAGE_SWITCHER = process.env.REACT_APP_LANGUAGE_SWITCHER === 'true';` i.e should be exported without the prefix `REACT_APP_`

Reply:
Also add `REACT_APP_LANGUAGE_SWITCHER` to `env.sample` and `docs/env.md`